lib/transaction.ex

```diff
@@ -88,6 +88,23 @@ defmodule Braintree.Transaction do
     end
   end
 
+  @doc """
+  Use a `transaction_id` and optional `amount` to settle the transaction.
+  Use this if `submit_for_settlement` was false while creating the charge using sale.
+
+  ## Example
+      {:ok, transaction} = Transaction.submit_for_settlement("123", %{amount: "100"})
+      transaction.status # 'settling'
```

Comment:
The status will be a binary, not a charlist. Use double quotes ("settling") here.